src/exporter/models/exporterStore.ts

```diff
@@ -43,13 +49,17 @@ export const exporterStore = types
         const snapshot = getSnapshot(self.searchParams);
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         const params: Record<string, unknown> = {};
+        params.packName = packInfo.packName;
+        params.minZoom = packInfo.minZoom;
+        params.maxZoom = packInfo.maxZoom;
         params.layers = [{layerType: 'raster', layerUrl: 'http://alex.rasterLayerUrl.com'}];
         params.bbox = [
           (snapshot.geojson as Polygon).coordinates[0], 
           (snapshot.geojson as Polygon).coordinates[2]
         ];
 
         try {
+          console.log('Fetch params--->',params);
```

Comment:
remove console
add issue to add app-logger if needed

Reply:
I leave it for purpose - to search easily where to MAKE final changes/integration